Bottom: bfc785d252677748f7512d33d3071fbfd3a5a994
Top:    729121d3fa01a451adc05b5e850a059edd49d552
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-16 21:26:17 +0300

Refresh of ctl-mgs.patch

---

diff --git a/ib-sock-int.h b/ib-sock-int.h
index ad39669..653cec8 100644
--- a/ib-sock-int.h
+++ b/ib-sock-int.h
@@ -7,6 +7,8 @@
 #include <rdma/ib_verbs.h>
 #include <rdma/rdma_cm.h>
 
+#define IB_CQ_EVENTS_MAX 128
+
 #define IB_ADDR_TIMEOUT 100
 #define IB_ROUTE_TIMEOUT 100
 
@@ -51,7 +53,6 @@ void sock_event_set(struct IB_SOCK *sock, unsigned int event)
 	wake_up(&sock->is_events_wait);
 }
 
-
 /* messages on wire */
 #define WIRE_ATTR	__attribute__((packed))
 
@@ -62,6 +63,11 @@ struct ib_hello {
 } WIRE_ATTR;
 
 
+/* mem.c */
+/* init function responsible to fill an number WR / SGE per socket*/
+int ib_sock_mem_init(struct IB_SOCK *sock);
+void ib_sock_mem_fini(struct IB_SOCK *sock);
+
 
 /* util.c */
 const char *ib_event_type_str(enum ib_event_type ev_type);
